web/src/ApiButton.tsx

```diff
@@ -38,7 +38,14 @@ const ApiButtonFormFooter = styled.div`
 `
 const ApiIconRoot = styled.div``
 export const ApiButtonLabel = styled.div``
-export const ApiButtonRoot = styled(ButtonGroup)`
+// MUI makes it tricky to get cursor: not-allowed on disabled buttons
+// https://material-ui.com/components/buttons/#cursor-not-allowed
+export const ApiButtonRoot = styled(ButtonGroup)<{ disabled?: boolean }>`
+  ${(props) =>
```

Comment:
we usually do this with CSS cascading?

Reply:
Yeah, that's how we've been doing it. I've been wanting to try out Styled Components props for a while, but hadn't figured out how to get them to play nicely with TypeScript.

I think the extra bit of typing might make them a bit nicer than class names - the compiler can catch issues and IDEs can jump between use to definition. If you've got a strong preference for sticking with class names, we can do that. Merging now - it's easy to change later.